client/src/templates/Challenges/classic/editor-tabs.tsx

```diff
@@ -47,7 +47,7 @@ class EditorTabs extends Component<EditorTabsProps> {
               data-cy={`editor-tab-${challengeFile.fileKey}`}
               onClick={() => toggleVisibleEditor(challengeFile.fileKey)}
             >
-              {`${challengeFile.name}.${challengeFile.ext}`}{' '}
+              <span>{`${challengeFile.name}.${challengeFile.ext}`}</span>{' '}
```

Comment:
Just wondering, what adding `span` do? I am missing what it helps with. 😞 
```suggestion
              <span>{`${challengeFile.name}.${challengeFile.ext}`}</span>
```